src/pymor/analyticalproblems/functions.py

```diff
@@ -372,7 +373,67 @@ def evaluate(self, x, mu=None):
         return np.prod([f(x, mu) for f in self.functions], axis=0)
 
 
-class BitmapFunction(Function):
+class DataFieldFunction(Function):
+    """Define a 2D |Function| via a matrix that represents a data field.
+
+    Parameters
+    ----------
+    data_field
+        Data field as a 2D-|NumPy array|.
+    bounding_box
+        Lower left and upper right coordinates of the domain of the data field.
+    range
+        A `data_field` value of p is mapped to `(p / scaling_factor) * range[1] + range[0]`,
+        where scaling_factor is set below.
+    scaling_factor
```

Comment:
Is this really needed? I'd suggest to just look at the minimum and maximum in `data_field` and scale the values according to the specified `range`.

Reply:
Yes that would be easier. I just took the same functionality as it already exists in the `BitmapFunction`. I will look into this.

Reply:
Hmm, thinking of multiple functions, you might have the situation that you want the same scaling for all functions even though the values in `data_field` do not cover the entire range. So maybe an optional `data_range` would be good, which is chosen instead to override `min(data_field)/max(data_field)` for the range of the unscaled data?